test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageControllerSpec.scala

```diff
@@ -69,6 +72,10 @@ class MultipleDisposalsTriageControllerSpec
 
   val mockUUIDGenerator = mock[UUIDGenerator]
 
+  val trustDisplay  = TrustAgentOrNormalDisplay(UserType.Organisation, Left(sample[TrustName]))
+  val agentDisplay  = TrustAgentOrNormalDisplay(UserType.Agent, Right(sample[IndividualName]))
+  val normalDisplay = TrustAgentOrNormalDisplay(UserType.Individual, Right(sample[IndividualName]))
```

Comment:
`individualDisplay` would be more appropriate

Reply:
Done.